resume-buddy/src/main/webapp/index.html

```diff
@@ -1,13 +1,58 @@
+ 
 <!DOCTYPE html>
 <html>
-   <head>
-      <meta charset="UTF-8">
-      <title>ResumeBuddy</title>
-      <link rel="stylesheet" href="style.css">
-      <script src="script.js"></script>
-   </head>
-
-   <body>
-      <p>Welcome to ResumeBuddy</p>
-   </body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>ResumeBuddy</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Noto+Sans+TC:wght@900&display=swap">
+    <script src="script.js"></script>
+  </head>
+  <body class="window">
+    <nav class="w3-sidebar w3-bar-block w3-white w3-collapse w3-top" style="z-index:3;width:250px" id="mySidebar">
```

Comment:
This `id` doesn't seem to be getting used in the CSS, so I think we should remove it from this PR and add it again if / when it is useful.